File: Techfest and the Queue.java
class Solution {
    public static long sum(long n) {
        long c=0;
        while(n%2==0) {
            c++;
            n/=2;
        }
        for(int i=3; i<=(int)Math.sqrt(n); i+=2) {
            while(n%i==0) {
                c++;
                n/=i;
            }
        }
        if(n>2) c++;
        return c;
    }
    public static long sumOfPowers(long a, long b) {
        // code here
        long ans = 0;
        for(long i=a; i<=b; i++) {
            ans += sum(i);
        }
        return ans;
    }
}

File: Techfest and the Queue.cpp
class Solution {
public:
    int sumOfPowers(int a, int b){
        // Code here
        int sum=0;
        for(int i=a;i<=b;i++)
        {
            int p=2;
            int j=i;
            while(sqrt(j)>=p)
            {
                if(j%p==0)
                {
                  sum++;
                  j/=p;
                }
                else p++;
            }
            if(i!=1)
            sum++;
        }
        return sum;
    }
};
